src/style.css

```diff
@@ -0,0 +1,183 @@
+/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+.nav__list {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 3px;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-transform: uppercase;
+  transition: color 0.3s;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  justify-content: center;
+  align-items: center;
+  font-size: 12px;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 15px;
+  bottom: 0;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin: 51px 129px 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 50px;
+  margin-bottom: 48px;
+  cursor: pointer;
+}
+
+.card__image img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin: 36px 16px 0;
+}
+
+.card__text {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  margin: 4px 84px 16px 16px;
+}
+
+.card__rating {
+  display: flex;
+}
+
+.card__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.card__star:nth-child(5) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__flex {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 17px 16px 23px;
+}
+
+.card__review {
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 400;
+  margin: 0 16px 16px;
+  color: #616070;
+}
+
+.card__priceBold {
+  font-size: 16px;
+  font-weight: 600;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: background-color 0.3s ease-in-out;
+  font-size: 14px;
+  font-weight: 700;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border: 0;
+  border-radius: 5px;
+  color: #fff;
+  text-transform: uppercase;
+  margin: 0 16px 16px;
+  padding: 0;
+  cursor: pointer;
+}
```

Comment:
```
.card__button {
  display: flex;
  justify-content: center;
  align-items: center;

  width: 166px;
  height: 40px;
  border: 0;
  border-radius: 5px;
  margin: 0 16px;
  padding: 0;

  background-color: #00acdc;
  
  font-size: 14px;
  font-weight: 700;
  text-transform: uppercase;
  color: #fff;
}
```

We don't need transition yet, but from design point of view it's good that you're thinking about it.

cursor: pointer; is default.

I changed order a bit, like we have learnt.